src/Util/Message.h

```diff
@@ -75,6 +76,7 @@ class Message {
     static CARTA::ImageProperties ImageProperties(std::string directory, std::string file, std::string hdu, int32_t file_id,
         CARTA::RenderMode render_mode, int32_t channel, int32_t stokes);
     static CARTA::ResumeSession ResumeSession(std::vector<CARTA::ImageProperties> images);
+    static CARTA::SetSpectralRequirements_SpectralConfig SetSpectralRequirements_SpectralConfig(const std::string& coordinate);
```

Comment:
My previous comment was misleading and I was looking at the wrong part of the code. I was thinking that:
```
CARTA::SetSpectralRequirements_SpectralConfig
```
should be:
```
CARTA::SetSpectralRequirementsSpectralConfig
```
To follow the naming conventions. I think you could revert this change if you want. A function name like `SetSpectralConfig` is totally fine.